libsolidity/ast/ASTJsonConverter.h

```diff
@@ -133,6 +133,7 @@ class ASTJsonConverter: public ASTConstVisitor
 	std::string location(VariableDeclaration::Location _location);
 	std::string contractKind(ContractDefinition::ContractKind _kind);
 	std::string functionCallKind(FunctionCallKind _kind);
+	std::string tokenKind(Token::Value _token);
```

Comment:
I'm wondering how to choose `toString()` or `tokenKind()`.  Maybe some comments in this file help.